x-pack/plugins/security_solution/common/detection_engine/schemas/request/add_prepackaged_rules_type_dependents.ts

```diff
@@ -92,6 +92,19 @@ export const validateTimelineTitle = (rule: AddPrepackagedRulesSchema): string[]
   return [];
 };
 
+export const validateThreshold = (rule: AddPrepackagedRulesSchema): string[] => {
+  if (rule.type === 'threshold') {
+    if (!rule.threshold) {
+      return ['when "type" is "threshold", "threshold" is required'];
+    } else if (rule.threshold.value <= 0) {
```

Comment:
Optionally you can use the io-ts type to verify the value is in bound -- either: 
`threshold.is(rule.threshold)` (as imported from `detection_engine/schemas/common/schemas.ts`)
-or-
`PositiveIntegerGreaterThanZero.is(rule.threshold.value)`